# The goal message
geometry_msgs/PoseStamped[] path
float64 velocity
float32 timeout_sec
float32 lookahead
float32 adaptive_lookahead
int32 drive_train_type
bool yaw_is_rate
float32 yaw
bool wait_on_last_task
---
# The result message
bool success
---
#feedback message
string current_point_on_path